### ISSUE TITLE
standardize XML for unknown response

### ISSUE THREAD

**Issue by crotwell** [FDSN/StationXML#16]
If a channel has no known response/sensitivity, the spec should define how to encode this. Usages might be both it is should be known but isn't, or where it is undefinable, like a textual LOG channel. Currently there are many options, all of which exist or have existed on servers.

Options:
1) Do not have a `<Response>` element on the channel
2) An empty `<Response>` element
3) An empty `<InstrumentSensitvity>` inside a `<Response>` (but invalid xml)
4) An obviously wrong `<InstrumentSensitivity>`like:
```
<Response>
  <InstrumentSensitivity>
    <Value>0.0</Value>
    <Frequency>0.0</Frequency>
    <InputUnits>
      <Name>NULL</Name>
      <Description>NULL</Description>
    </InputUnits>
    <OutputUnits>
      <Name>COUNTS</Name>
      <Description>Digital Counts</Description>
    </OutputUnits>
  </InstrumentSensitivity>
</Response>
```

For a client, having to deal with all of these variations as effectively meaning the same thing is kind of a pain and so would be really useful if the spec and/or schema would pick one representation of unknown/undefined and mandate it.

**Comment by PetrColinSky** [comment]
Yes! Vey important idea! Moreover, ObsPy easily gets over empty <code>Response</code> element without any warning producing "deconvolved" signal which is the same as the original as nothing has happened, but the user may not notice it. Way worse than no information is wrong information. I mean - when there is no response, in the end it is easy to notice it. If there is wrong response, one gets lost. But still, I am on to define how to handle "no-response" case. At least it will be clear, that it is not missing by mistake, but deliberately.

**Comment by rcasey-iris** [comment]
Current behavior of IRIS FDSNWS Station service is to present an empty <Response> element, such as is the case with an ACE channel for a TA station:

```
    <Channel code="ACE" locationCode="" startDate="2013-06-12T00:00:00.0000" endDate="2015-05-20T20:15:00.0000" restrictedStatus="open">
     <Comment>
      <Value>0100000B65267F35</Value>
     </Comment>
     <Latitude>41.541302</Latitude>
     <Longitude>-75.432098</Longitude>
     <Elevation>471</Elevation>
     <Depth>0</Depth>
     <Azimuth>0</Azimuth>
     <Dip>0</Dip>
     <Type>HEALTH</Type>
     <SampleRate>0E00</SampleRate>
     <ClockDrift>0E00</ClockDrift>
     <Sensor>
      <Description>Quanterra 330 Linear Phase Composite</Description>
     </Sensor>
     <Response>
     </Response>
    </Channel>
```

**Comment by crotwell** [comment]
I think I prefer option 2, the IRIS way, an empty `<response></response>`.

Option 1 is bad because you can't tell a missing response from never asking for it, ie fdsnws station with level=channel does not have a `<response>` because it wasn't asked for.
Option 3 is invalid, and we shouldn't recommend invalid xml.

Option 4 is ok I guess, and may help with the concern about obspy.

**Comment by crotwell** [comment]
If we go with option 2, PR #117 fixes this.

**Comment by PetrColinSky** [comment]
Yes, option 2 sounds the best. Option 4 can be misleading when looking at the xml file visually, as it may look like something is in the response. It may not be obvious to everybody, that it is wrong. PR #117 looks perfect.

**Comment by crotwell** [comment]
fixed in PR #117
